Source/ARTPushActivationEvent.h

```diff
@@ -41,10 +41,10 @@ NS_ASSUME_NONNULL_BEGIN
 @interface ARTPushActivationEventGotPushDeviceDetails : ARTPushActivationEvent
 @end
 
-@interface ARTPushActivationEventGotUpdateToken : ARTPushActivationEvent
+@interface ARTPushActivationEventGotDeviceRegistration : ARTPushActivationEvent
 @end
 
-@interface ARTPushActivationEventGettingUpdateTokenFailed : ARTPushActivationErrorEvent
+@interface ARTPushActivationEventGettingDeviceRegistrationFailed : ARTPushActivationErrorEvent
```

Comment:
I think we can shorten it as `ARTPushActivationEventDeviceRegistrationFailed `

Reply:
But `GettingUpdateTokenFailed` has been renamed to `GettingDeviceRegistrationFailed`, so I think it's better to leave it that way for consistency.

Reply:
Right. We had this discussion in Slack. It's kinda late to rename everything at this point.